lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -67,7 +67,11 @@ function handleCreateDeviceGroup(req, res, next) {
     for (var i = 0; i < req.body.services.length; i++) {
         req.body.services[i].service = req.headers['fiware-service'];
         req.body.services[i].subservice = req.headers['fiware-servicepath'];
+        req.body.services[i].type = req.body.services[i]['entity_type'];
         req.body.services[i].internalAttributes = req.body.services[i]['internal_attributes'];
+
+        //delete req.body.services[i]['entity_type'];
+        //delete req.body.services[i]['internal_attributes'];
```

Comment:
I didn't. I meant to add them, but I remove them to test a couple of things. I'll uncomment them now.